src/sentry/demo/demo_start.py

```diff
@@ -250,3 +259,13 @@ def _get_one_transaction_name(project: Project):
         referrer="sandbox.demo_start._get_one_transaction_name",
     )
     return result["data"][0]["transaction"]
+
+
+def check_strings_similar(error_type, group):
+    type = group.data["metadata"].get("type")
+    title = group.data["metadata"].get("title")
+
+    error = type or title
```

Comment:
@roggenkemper it would be good if we could check both fields. So even if the `type` exists but doesn't match, we could still check the title.